lessons/Early-Math/absolute-value-and-opposites/langs/en-us/index.adoc

```diff
@@ -0,0 +1,150 @@
+= Absolute Value and Opposites
+
+@description{Students use Circles of Evaluation to analyze the effect of negation and absolute value.}
+
+@lesson-prereqs{distributive-property}
+
+@keywords{Opposite, Absolute Value}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Define absolute value as a number's distance from zero on the number line.
+* Recognize that two numbers are referred to as "opposites" if they are the same distance from zero.
+* Use Circles of Evaluation to analyze expressions with opposite and absolute value.
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Circles of Evaluation to think about opposites and absolute value.
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Preparation
+|
+* Draw or project a number line, which will be the focal point of a lot of the conversation for today's lesson.
+
+| Key Points For The Facilitator
+|
+* The difference between absolute value and opposite can be subtle and confusing for students. Sometimes, given an input, they produce the same output - but at other times, their outputs differ.
+|===
+
+== Finding Opposites @duration{20 minutes}
+
+=== Overview
+
+Students discover what it means for numbers to be @vocab{opposites}, and explore the impact of the `negate` in Circles of Evaluation.
+
+
+=== Launch
+
+Instruct students to draw a number line that spans from @math{-10} to @math{10}, or draw/project one at the front of the room.
+
+[.lesson-instruction]
+- What number is the same distance from zero as @math{8}?
+** _@math{-8}_
+- What number is the same distance from zero as @math{-3}?
+** _@math{3}_
+- The pairs of numbers we've just identified are called @vocab{opposites} - two numbers that are the same distance from zero on the number line, with one negative and one positive.
+- What are some other pairs of opposites that you can identify?
+** _Possible responses: @math{6.5} and @math{-6.5}; @math{4} and @math{-4}; @math{\frac {7}{3}} and @math{- \frac {7}{3}}._
+- Are there any numbers on this line that don't have an opposite?
+** _Students may guess that zero does not have an opposite - however zero is its own opposite!_
```

Comment:
I'm not sure that this is a point worth focusing on... it just feels like it's meant to trick students without actually deepening understanding. Why does it matter whether zero has no opposite or is its own opposite? The only answer I can come up with is if our understanding of opposites is that they are additive inverses... which is a concept you haven't introduced.